resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesUtils.scala

```diff
@@ -89,26 +56,16 @@ private[spark] object KubernetesUtils {
   def getOnlyRemoteFiles(uris: Iterable[String]): Iterable[String] = {
     uris.filter { uri =>
       val scheme = Utils.resolveURI(uri).getScheme
-      scheme != "file" && scheme != "local"
+      scheme != "local"
```

Comment:
Change seems unnecessary for now, since `file` is still forbidden by our submission code.

Reply:
Unnecessary, but it should not be allowed in our Utils as we can't resolve the location without `SparkFiles.get()`which makes that logic useless. No?

Reply:
Actually is this function still needed?

Reply:
At the moment is not being used. Should it be removed then?

Reply:
Remove this for now.

Reply:
Done.